src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
     <input
+      class="bar big-bar"
       type="text"
-      data-qa="keypress"
+      data-qa="big keypress"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
It's a good practice to include a label for input elements to improve accessibility. Consider adding a `<label>` element associated with this input field.